src/script/interpreter.cpp

```diff
@@ -1142,24 +1142,24 @@ class CTransactionSignatureSerializer {
 
 uint256 GetPrevoutHash(const CTransaction& txTo) {
     CHashWriter ss(SER_GETHASH, 0);
-    for (unsigned int n = 0; n < txTo.vin.size(); n++) {
-        ss << txTo.vin[n].prevout;
+    for (const auto& txIn : txTo.vin) {
```

Comment:
@luke-jr Likely I'm missing something here, but `txTo.vin` is `std::vector<CTxIn>` and not `CTransaction`. Shouldn't `txIn` then indicate `CTxIn`. What am I missing? :-)